SEQLinkage/Utils.py

```diff
@@ -120,7 +120,8 @@ def __del__(self):
                     remove_tree(os.path.join(where, fn))
                 except:
                     pass
-        tmp = LockedTempDir(tempfile.mkdtemp(prefix='{}_tmp_'.format(self.proj), dir = where))
+        #tmp = LockedTempDir(tempfile.mkdtemp(prefix='{}_tmp_'.format(self.proj), dir = where))
```

Comment:
why do we have to comment this out? I think it is safer to create project specific cache folders?

Reply:
The tmp folder is different each time we run it, which makes no way to using the results from last run. The cache folder is still there, but with specific name.